docs/introduction.tex

```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
+a better understanding of the dependence between various components 
+in the system and the effects of changes on the system. 
+Their goal is to assist in evaluating and improving potential 
+strategies for nuclear power development in terms of improving waste 
+management, economic competitiveness, etc \cite{yacout_modeling_2005}.   
+
+One of the major functionalities of a \gls{NFC} simulator is its 
+ability to transmute nuclear fuel in a reactor based on reactor 
+conditions such as burnup, enrichment, etc. 
+The transmutation results impact the accuracy of the \gls{UNF} 
+composition, and thus the capability of using the \gls{NFC} to 
+analyze the impact of a \gls{NFC} on variables such as waste profile.  
+
+Current fuel cycle simulator codes include \Cyclus, ORION, DYMOND 
+and VISION. 
+Each of these NFC simulators obtain transmutation results using 
+different methods. 
+Some of these codes have multiple methods for obtaining 
+transmutation results. 
+There is essentially two overall types of methods to obtain transmutation 
+results: recipe-based method and library-based method. 
+The recipe-based method is a low fidelity method, while the library-based method is 
+a high fidelity method. 
+In the recipe-based method, direct neutronics calculations are not performed 
+within the model, it is done externally \cite{yacout_vision_2006}. 
+The resulting fresh and spent fuel compositions for specific parameters 
+is entered directly into the \gls{NFC} model \cite{sunny_transition_2015}. 
+In the library-based method,  the \gls{NFC} code dynamically calculates depleted 
+fuel recipes by interpolating reactor data libraries generated by
+reactor physics burnup calculation code such as SERPENT 
+\cite{leppanen_serpent_2013}, Origen \cite{croff_users_1980}, etc. 
+Table \ref{tab:nfc_code} shows a simple breakdown of the 
+transmutation methods available for each \gls{NFC} code. 
+
+\begin{table}[h]
+    \centering
+    \label{tab:nfc_code}
+    \begin{tabular}{lrrr}
+        \hline
+        \gls{NFC} code & Transmutation Methods Available \\
+        \hline
+        \Cyclus & Recipe-based and Library-based\\
+        ORION & Recipe-based and Library-based\\
+        DYMOND & Recipe-based  \\
+        VISION & Recipe-based  \\
+        \hline
+    \end{tabular}
+    \caption{\gls{NFC} Methods for transmutation in reactor module}
+\end{table}
+
+\Cyclus has three methods to obtain transmutation results: \Cycamore 
+recipe reactor \cite{huff_extensions_2014} CyBORG 
+\cite{skutnik_cyborg_2016} and Bright-lite \cite{flanagan_brightlite_2014}.  
+The \Cycamore recipe reactor accepts a fresh and spent fuel recipe that is 
+defined by the user. 
+CyBORG calls the Origen solver \gls{API} directly to calculate depleted fuel
+compositions, which then becomes the output recipe for the CyBORG reactor 
+in \Cyclus \cite{skutnik_cyborg_2016}. 
+Bright-lite uses libraries that represent a specific reactor design to 
```

Comment:
this is quite unclear. How does it calculate the burnup? Maybe it helps to illustrate what the user puts in and how the algorithm uses the user inputs, and what the algorithm outputs